packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -29,19 +33,86 @@ String userAppType = ''; // dimension1
 String userBuildType = ''; // dimension2
 String userPlatformType = ''; // dimension3
 
-void computeApplicationState(
-  bool isAnyFlutterApp,
-  bool isFlutter,
-  bool isWebApp,
-  bool isProfile,
-  bool isAndroid,
-) {
+String devtoolsPlatformType = ''; // dimension4 MacIntel/Linux/Windows/Android_n
+String devtoolsChrome = ''; // dimension5 Chrome/n.n.n  or Crios/n.n.n
+const String devtoolsVersion = devtools.version; //dimension6 n.n.n
+
+bool get isDimensionsComputed =>
+    (userAppType.length +
+        userBuildType.length +
+        userPlatformType.length +
+        devtoolsPlatformType.length +
+        devtoolsChrome.length) >
+    0;
+
+/// Computes the DevTools application. Fills in the devtoolsPlatformType and
+/// devtoolsChrome.
+void computeDevToolsState() {
```

Comment:
nit: 
`computeAnalyticsState`
or
`computeGTagsState`
would convey the intent more.
Also consider that when we use Flutter Web, the word State means something pretty specific so might be good to avoid the word state when it doesn't mean Flutter State.

Reply:
I think this is more precise:

_computeDevToolsCustomGTagsData
computeUserApplicationCustomGTagData

Reply:
sgtm